scripts/bash/buildInternalDependencies.sh

```
@@ -1,56 +1,25 @@
-    echo "> ${CONCURRENTLY_BIN} -m $CONCURRENT_BUILD_BATCH_SIZE --names \"${build_prefixes[*]}\" -k ${build_commands[*]}"
-    echo ""
-    eval   "${CONCURRENTLY_BIN} -m $CONCURRENT_BUILD_BATCH_SIZE --names \"${build_prefixes[*]}\" -k ${build_commands[*]}"
-fi
+REPO_ROOT=$DIR/../..
+
+# Convert 'foo-bar baz' → '@tupaia/{foo-bar,baz}'.
+# Assume INTERNAL_DEPS is a valid space-separated list of package names, and use
+# Bash parameter expansion to replace spaces with commas.
+INTERNAL_DEPS=$("$DIR/getInternalDependencies.sh")
+INTERNAL_DEPS_CSV=${INTERNAL_DEPS// /,}
+PATTERN=@tupaia/{$INTERNAL_DEPS_CSV}
```

Shellcheck (0.10.0)

[warning] This { is literal. Check expression (missing ;/\n?) or quote it.

(SC1083)